cache/statusCalculation.js

```diff
@@ -70,6 +71,26 @@ function setRanges(ranges, aspName) {
   .execAsync();
 }
 
+/**
+ * Add redis commands to set keys for the given ranges
+ *
+ * @param  {Array<Object>} ranges - sorted, non-overlapping list of aspect ranges
+ * @param  {String} aspName - aspect name
+ * @param  {Array} cmds - Redis commands
+ * @returns {Promise}
+ */
+function addRangesCmds(ranges, aspName, cmds) {
```

Comment:
same comment as earlier about opaque "side-effects" although this one feels more forgivable than the other one